astropy/units/quantity.py

```diff
@@ -1862,7 +1894,11 @@ def nansum(self, axis=None, out=None, keepdims=False):
             return self._wrap_function(np.nansum, axis,
                                        out=out, keepdims=keepdims)
     else:
+        # TODO: deprecate this method? It is not on ndarray, and we do not
```

Comment:
Is there a follow-up issue for this?

Reply:
No, it is not urgent and has been around forever. So either I do it very soon or just whenever someone encounters this.......